newIDE/app/src/ObjectEditor/Editors/SpriteEditor/DirectionTools.js

```diff
@@ -185,6 +185,7 @@ export default class DirectionTools extends Component<Props, State> {
                 this.setState({ timeBetweenFrames: text })
               }
               isLooping={direction.isLooping()}
+              hideAnimationLoader // No need to show a loader in the Direction Tools.
```

Comment:
It is true when on Desktop because the resources are local, but should we show the loader for cloud projects?

Reply:
We could, but as this tool is not the "external facing" animations, like in the Asset Store, and more of a "internal tool" to work on your animation, I'm not sure it adds value.